index.bs

```diff
@@ -7,8 +7,10 @@ Group: SACG
 URL: https://w3c.github.io/speech-api/
 Repository: w3c/speech-api
 Shortname: speech-api
-Editor: Glen Shires (Google)
-Editor: Hans Wennborg (Google)
+Editor: Andre Natal (Mozilla)
+Editor: Philip Jägenstedt (Google)
+Former Editor: Glen Shires (Google)
```

Comment:
@gshires if you do still intend to do any work on this spec please let me know and I'll revert the "Former" bit here. This seems to reflect the current state of things, but I'm not trying to make more work for myself if not necessary :)

Reply:
Searching for something else in my inbox I came across the thread about setting up this repo where you said "I'd like to remain as an editor" so I'll revert this bit as well. I can't find Hans on GitHub at all so will leave him as former.